src/servers/Cargo.toml

```diff
@@ -27,7 +27,8 @@ arrow-schema.workspace = true
 async-trait = "0.1"
 auth.workspace = true
 axum = { workspace = true, features = ["multipart"] }
-axum-macros = "0.3.8"
+axum-extra = { version = "0.10", features = ["typed-header"] }
+axum-macros = "0.4"
```

Comment:
sugg: use workspace level dependency?

Reply:
Now only the servers crate uses them. I'm not sure if we should move it to the workspace level.

Reply:
https://github.com/GreptimeTeam/greptimedb/blob/bump-df-202412/Cargo.toml#L98-L99
It's better to put them together if they are meant to be updated together.

Reply:
fixed in https://github.com/GreptimeTeam/greptimedb/pull/5417/commits/3a28240aa8460b34d5b76f2b6c2ce7b60ce7c667